src/components/NcChip/NcChip.vue

```diff
@@ -232,6 +234,13 @@ const onClose = () => {
 		overflow: hidden;
 		height: var(--chip-size);
 		width: var(--chip-size);
+
+		&--empty {
+			flex: 1;
+			width: 0;
+			// Add some more space to the border
+			margin-inline-end: calc(2 * var(--default-grid-baseline));
```

Comment:
Hmm, because of this, I guess.

But then I'd add the class on the chip itself to add the margin. Otherwise there is always `__icon` in the layout even when there is no actual icon but just an element for margin.